content/en/security/cloud_workload_security/setup.md

```diff
@@ -40,7 +40,7 @@ There are four types of monitoring that the Datadog Agent uses for Cloud Workloa
 
 ## Requirements
 
-* Datadog Agent >= 7.27.0
+* Datadog Agent >= 7.4.0
```

Comment:
Should it be 7.40.0 ? Do you have a link to the discussion about this bump ?

Reply:
I can chime in as I've been working on this with John. There's been a lot of discussion as to which version should be listed as the minimum required version for CWS. I should note that on the [security profiles doc](https://docs.datadoghq.com/security/cloud_workload_security/security_profiles), we list 7.44 as the required version.

I suggest we hold off on making this change until after John returns next week. I'll follow-up with him to confirm what he wants to do to ensure we're consistent across all of the CWS docs.